### PR TITLE
Update timetable url to point to new endpoint

### LINKED ISSUE #143: [BUG]: Fails to parse time table for first year
### Time table Document

1st year, Phy sem
```<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
    "http://www.w3.org/TR/html4/loose.dtd">
















<html>
    <head>
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <title>Time Table View</title>
        <link rel="StyleSheet" href="../css/style1.css">
    </head>
    <body>
        <form name="frm" action="secwiseview.jsp?action=first" method="post">
              <center><font color="#000099"><b>Your TimeTable for AUTUMN 2024-2025</b></font></center>
                        <table><tr><td><table border="1" cellpadding="0" cellspacing="0" >
                                        <tr><td class=tableheader nowrap style='padding-top:5px;padding-bottom:5px;padding-left:7px;padding-right:7px'>Day Name</td>
                                            <TD class=tableheader nowrap style='padding-top:5px;padding-bottom:5px;padding-left:7px;padding-right:7px'>8:0:AM-8:55:AM</TD>
<TD class=tableheader nowrap style='padding-top:5px;padding-bottom:5px;padding-left:7px;padding-right:7px'>9:0:AM-9:55:AM</TD>
<TD class=tableheader nowrap style='padding-top:5px;padding-bottom:5px;padding-left:7px;padding-right:7px'>10:0:AM-10:55:AM</TD>
<TD class=tableheader nowrap style='padding-top:5px;padding-bottom:5px;padding-left:7px;padding-right:7px'>11:0:AM-11:55:AM</TD>
<TD class=tableheader nowrap style='padding-top:5px;padding-bottom:5px;padding-left:7px;padding-right:7px'>12:0:PM-12:55:PM</TD>
<TD class=tableheader nowrap style='padding-top:5px;padding-bottom:5px;padding-left:7px;padding-right:7px'>2:0:PM-2:55:PM</TD>
<TD class=tableheader nowrap style='padding-top:5px;padding-bottom:5px;padding-left:7px;padding-right:7px'>3:0:PM-3:55:PM</TD>
<TD class=tableheader nowrap style='padding-top:5px;padding-bottom:5px;padding-left:7px;padding-right:7px'>4:0:PM-4:55:PM</TD>
<TD class=tableheader nowrap style='padding-top:5px;padding-bottom:5px;padding-left:7px;padding-right:7px'>5:0:PM-5:55:PM</TD>
</TR>
<TR>
<TD class='tableheader' valign='top' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=1 >Mon</TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=1><b>MA11003<BR>NR111</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=1><b>MA11003<BR>NR111</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=1><b>EE11003<BR>NR212</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=1><b>CS10003<BR>NR111</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=1><b>&nbsp;</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=3><b>ME29201<BR>IN DEPT.</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=1><b>&nbsp;</b></TD>
</TR>
<TR>
<TD class='tableheader' valign='top' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=1 >Tue</TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=1><b>CS10003<BR>NR111</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=1><b>CS10003<BR>NR111</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=1><b>&nbsp;</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=1><b>&nbsp;</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=1><b>MA11003<BR>NR111</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=3><b>PH19003<BR>JC Bose Lab.Complex</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=1><b>&nbsp;</b></TD>
</TR>
<TR>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=3><b>ME29201<BR>IN DEPT.</b></TD>
</TR>
<TR>
<TD class='tableheader' valign='top' style='padding-top:5px;padding-bottom:5px' rowspan=3 colspan=1 >Wed</TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=3 colspan=1><b>EE11003<BR>NR212</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=3 colspan=1><b>EE11003<BR>NR212</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=3 colspan=3><b>ME29201<BR>IN DEPT.</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=3><b>ME29201<BR>IN DEPT.</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=3 colspan=1><b>&nbsp;</b></TD>
</TR>
<TR>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=3><b>CS19003<BR>CIC LAB.</b></TD>
</TR>
<TR>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=3><b>CS19003<BR>CIC. LAB.</b></TD>
</TR>
<TR>
<TD class='tableheader' valign='top' style='padding-top:5px;padding-bottom:5px' rowspan=3 colspan=1 >Thur</TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=3 colspan=1><b>&nbsp;</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=3 colspan=1><b>&nbsp;</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=3 colspan=3><b>ME29201<BR>IN DEPT.</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=3><b>ME29201<BR>IN DEPT.</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=3 colspan=1><b>EE11003<BR>NR212</b></TD>
</TR>
<TR>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=1><b>&nbsp;</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=1><b>PH11003<BR>NR211</b></TD>
</TR>
<TR>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=1><b>&nbsp;</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=1><b>&nbsp;</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=1><b>PH11003<BR>NR211</b></TD>
</TR>
<TR>
<TD class='tableheader' valign='top' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=1 >Fri</TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=1><b>&nbsp;</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=1><b>&nbsp;</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=3><b>ME29201<BR>IN DEPT.</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=1><b>PH11003<BR>NR211</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=1><b>PH11003<BR>NR211</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=1><b>&nbsp;</b></TD>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=2 colspan=1><b>MA11003<BR>NR111</b></TD>
</TR>
<TR>
<TD  align='center' style='padding-top:5px;padding-bottom:5px' rowspan=1 colspan=3><b>ME29201<BR>IN DEPT.</b></TD>
</TR>

                                    </table></td></tr></table><input type=hidden name=semester id=semester value="AUTUMN">
                        <input type=hidden name=session id=session value="2024-2025">

                    </form></td></tr></table></body>
</html>
```

### Probable cause

Multiple classes in one slot (actual timetable shared on teams).

### Observed Behaviour

No classes are added but insti holidays are.

### PR DESCRIPTION
**Pull request by dipamsen**
Fixes #143 

Origin of error: Since this semester, ERP serves up the timetable on a new endpoint (`/student_timetable.jsp`). It just so happens that the old endpoint (`view_stud_time_table.jsp`) still works for everyone except first year students, for whom only the new endpoint works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,8 +51,6 @@
 
 Gets your timetable from ERP and adds it to your Google Calendar or gives you an ICS file which you can add in any common calendar application.
 
-> **Note** All updates to this repo should reflect, with appropriate refactorisation, in [gyft-serve](https://github.com/metakgp/gyft-serve/)
```
Comment (harshkhandeparkar) [inline README.md:54]: I think we should keep this PR focused on the relevant changes. It would be great if you open a separate PR updating the documentation since it is a bit out of date.

Comment (dipamsen) [reply README.md]: reverted back the change

Comment (harshkhandeparkar) [reply README.md]: Can you rebase and make the commit vanish?

**Review by proffapt — APPROVED** [review]
https://github.com/metakgp/gyft/issues/143#issuecomment-2386764059

LGTM.
